packages/@aws-cdk-testing/framework-integ/test/aws-rds/test/integ.temp.js.snapshot/EnhancedMonitoringClusterTestStack.template.json

```diff
@@ -0,0 +1,858 @@
+{
+ "Resources": {
+  "Vpc8378EB38": {
+   "Type": "AWS::EC2::VPC",
+   "Properties": {
+    "CidrBlock": "10.0.0.0/16",
+    "EnableDnsHostnames": true,
+    "EnableDnsSupport": true,
+    "InstanceTenancy": "default",
+    "Tags": [
+     {
+      "Key": "Name",
+      "Value": "EnhancedMonitoringClusterTestStack/Vpc"
+     }
+    ]
```

Comment:
what is this `integ.temp.js` test? I don't see any integration testss attach to it.

Reply:
I'm sorry.. I've removed it.